ci/maven.yml

```diff
@@ -24,3 +29,7 @@ jobs:
         cache: maven
     - name: Build with Maven
       run: mvn -B package --file pom.xml
+    
+    # Uploads the full dependency graph to GitHub to improve the quality of Dependabot alerts this repository can receive
+    - name: Update dependency graph
+      uses: advanced-security/maven-dependency-submission-action@v1
```

Comment:
Kindly use commit sha instead of version. This is the latest tag:`f0bf8f8f5feaaf5fc8d6e7e315bf877e464b1d9a`

Reply:
Is this a mandatory step or optional?

Reply:
Optional

Reply:
- [ ]

Reply:
Updated to use a tag, and marked the step as optional in the comments.